src/index.ts

```diff
@@ -0,0 +1 @@
+export {default} from './useA11yBlockLink';
```

Comment:
Totally up to you, if this package will only export the single hook, you could avoid the indirection here and inline the hook in the index file + remove the `useA11yBlockLink` folder 👍